jboss-jakartaee-web/pom.xml

```diff
@@ -35,7 +35,7 @@
 
   <issueManagement>
     <system>jira</system>
-    <url>http://jira.jboss.com/jira/browse/JBEE</url>
+    <url>http://issues.redhat.com/jira/browse/JBEE</url>
```

Comment:
@whitingjr assuming this investigation would continue, please could you  the URL https too please (and the one below) but this JBEE project doesn't seem to exist https://issues.redhat.com/jira/browse/JBEE and I am not sure if we want to try to reinstate it? Maybe people should raise issues in this GitHub project instead (or WFLY instead)? I don't really know - @bstansberry - do you have any suggestions?

Reply:
https://issues.redhat.com/browse/JBEE seems to be the correct link now

Reply:
@whitingjr please can you update this line in here to https://issues.redhat.com/browse/JBEE?

Reply:
@tomjenkinson yes that was done [in this commit](https://github.com/jboss/jboss-jakartaee-specs/pull/3/commits/0f574da9004e586c63785d2c465103f9d9a73de5).

Reply:
@whitingjr I will have a look at the commit log but when I look at https://github.com/jboss/jboss-jakartaee-specs/pull/3/files it is not showing as being changed to https://issues.redhat.com/browse/JBEE whereas the pom.xml in the root is showing that change.

Reply:
@whitingjr the commit you mentioned (https://github.com/jboss/jboss-jakartaee-specs/pull/3/commits/0f574da9004e586c63785d2c465103f9d9a73de5) changes the root pom.xml but the URL I am referring to is in `jboss-jakartaee-web/pom.xml`

Reply:
thanks for pointing that out. The additional file has been updated.